lib/Compilation.js

```diff
@@ -517,7 +517,8 @@ BREAKING CHANGE: Asset processing hooks in Compilation has been merged into a si
 			additionalAssets: createProcessAssetsHook(
 				"additionalAssets",
 				Compilation.PROCESS_ASSETS_STAGE_ADDITIONAL,
-				() => []
+				() => [],
+				"dep_webpack_compilation_additional_assets".toUpperCase()
```

Comment:
It is soft deprecation without warnings

Reply:
Only hooks will be removed in webpack 6 show those warnings?

Reply:
in webpack@6 we will emit deprecation for this

Reply:
Got it.